README.md

```diff
@@ -109,6 +109,21 @@ To install them on Debian or Ubuntu, run
 sudo apt-get install pkg-config clang libwayland-dev libxkbcommon-x11-dev libvulkan-dev
 ```
 
+## Project structure
+
+This diagram gives an idea what the Xilem project is built on:
+
+![docs/assets/xilem-layers.svg]
+
+On a very coarse level, Xilem is built directly on top of xilem_core and Masonry, both of which are crates in this repository.
+
```

Comment:
When introducing this new section, it probably makes sense to link to `ARCHITECTURE.md` (which needs some rework as well) for more details?